sourcemash/frontend/static/js/views/profile_view.js

```diff
@@ -18,5 +19,24 @@ Sourcemash.Views.ProfileView = Backbone.View.extend({
   deleteUser: function() {
     this.model.destroy();
     mixpanel.track("Deleted account");
+  },
+
+  toggleShowUnsubscribedContent: function() {
+      if (this.model.get('show_unsubscribed_content')) {
+          this.model.save({'show_unsubscribed_content': false}, {success: this.showContentToggled});
```

Comment:
Get input from checkbox, without the not? We do want to send the API the input data after all.